src/extension/inline/Transition.re

```diff
@@ -0,0 +1,167 @@
+type styleT;
+
+external makeStyle :
+ azimuth::string? =>
```

Comment:
lol I appreciate the thoroughness here, but I don't think we should include obsolete attributes unless absolutely necessary.